make-release.sh

```diff
@@ -91,19 +92,19 @@ gitClone() {
   fi
 }
 
-gitBranch() {
+checkoutVersionBranch() {
   # Handle the version branch: create it or update it.
   # Check if the branch exists, locally or on the remote.
   EXISTING_BRANCH=0
   git fetch
-  EXISTING_BRANCH=$(git ls-remote --heads origin "${TARGET_BRANCH}")
+  EXISTING_BRANCH=$(git ls-remote --heads origin "${BUGFIX_BRANCH}")
   case ${EXISTING_BRANCH} in
     "") echo "[INFO] Creating new branch: ${TARGET_BRANCH} from branch: ${MAIN_BRANCH}."
       git checkout "${MAIN_BRANCH}"
-      git checkout -b "${TARGET_BRANCH}"
+      git checkout -b "${BUGFIX_BRANCH}"
       ;;
-    *) echo "[INFO] Updating branch: ${TARGET_BRANCH} from branch: ${MAIN_BRANCH}."
-      git checkout "${TARGET_BRANCH}"
+    *) echo "[INFO] Updating branch: ${BUGFIX_BRANCH} from branch: ${MAIN_BRANCH}."
```

Comment:
We should not rebase the branch on main. I believe only the message displayed is erroneous.


```suggestion
    *) echo "[INFO] Updating branch: ${BUGFIX_BRANCH}."
```